modular_skyrat/modules/ipc_printing/code/ipc_printing_techweb.dm

```diff
@@ -0,0 +1,16 @@
+/datum/techweb_node/ipc_parts
+	id = "ipc_parts"
+	display_name = "IPC Organs"
+	description = "Allows Exosuit Fabricators to build IPC organs."
+	prereq_ids = list("basic_medical")
```

Comment:
That means it'll be available to be researched by default. I'd probably move that to be after something as part of the robotics research tree.

Reply:
yeah, makes sense. i'd originally meant it to have the same pre-req as cyber organs but read the labels wrong at the time. i'll take another look at them.